test/stat/stat_test.go

```diff
@@ -96,6 +105,18 @@ func TestCliStatForConduitNamespace(t *testing.T) {
 				TestHelper.GetConduitNamespace(): "4/4",
 			},
 		},
+		{
+			args: []string{"stat", "po", "-n", TestHelper.GetConduitNamespace(), "--to", "au/prometheus.conduit.svc.cluster.local:9090"},
+			expectedRows: map[string]string{
+				controllerPod: "1/1",
+			},
+		},
+		{
+			args: []string{"stat", "au", "-n", TestHelper.GetConduitNamespace(), "--to", "au/prometheus.conduit.svc.cluster.local:9090"},
+			expectedRows: map[string]string{
+				"prometheus.conduit.svc.cluster.local:9090": "-",
+			},
+		},
```

Comment:
I think this test case would be more valuable if it were the inverse of the previous test. Specifically, the previous test requests stats from pods to a specified authority, and I think this one should test stats from authorities to a specified pod. I'd rewrite this test case as:

```
		{
			args: []string{"stat", "au", "-n", TestHelper.GetConduitNamespace(), "--to", "po/" + prometheusPod},
			expectedRows: map[string]string{
				prometheusAuthority: "-",
			},
		},
```